worlds/openrct2/data/location_info.py

```diff
@@ -0,0 +1,812 @@
+location_info = {
+    "all_locations":[

```

Comment:
instead of having to hardcode these by hand i recommend just use a list comphrehension
```
[
location_info = {
    "all_locations":
    f"{color}{i}" 
    for i in range(100)
    for color in ("White", "Black", "Green", "Blue", "Yellow", "Gold", "Silver", "Celadon", "Pink")
]
}
```